build-logic-settings/settings.gradle.kts

```diff
@@ -22,7 +22,4 @@ dependencyResolutionManagement {
     }
 }
 
-include("allprojects-plugin")
-include("testfiltering-plugin")
-include("cc-experiment-plugin")
```

Comment:
It seems like you'd still need to delete these directories and the build files inside, right?

Reply:
You mean locally? Why we need to delete them if they are not included?

Reply:
Oh, build script, yes

Reply:
Done.